pypeman/channels.py

```diff
@@ -1,7 +1,7 @@
 import asyncio
-import os
 import uuid
 import logging
+import pathlib
```

Comment:
why not use 
`from pathlib import Path`?

This is suggested in almost all tutorials